src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -317,8 +317,11 @@ export function registerTerminalActions() {
 					return;
 				}
 				c.service.setActiveInstance(instance);
+				await focusActiveTerminal(instance, c);
+			}
+			else {
+				await c.groupService.showPanel(true);
```

Comment:
I think line 323 can be removed in favor of 320 now

Reply:
Applied it now